ajax/libs/jplist/package.json

```diff
@@ -17,8 +17,48 @@
     "type": "git",
     "url": "https://github.com/no81no/jplist"
   },
-  "license": "MIT",
-  "authors": [
-    "Miriam Zusin <no81no@gmail.com>"
-  ]
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/no81no/jplist.git",
+    "basePath": "",
```

Comment:
We should set basePath to `"dist"` here and remove the `dist/` prefix in filemap. If you want to include whole `dist` folder, please use single rule `"**/*"`, thanks.